aspnetcore/Dockerfile.template

```diff
@@ -0,0 +1,23 @@
+﻿FROM mcr.microsoft.com/dotnet/aspnet:7.0 AS base
+WORKDIR /app
+ENV ASPNETCORE_URLS=http://+:8080
+EXPOSE 8080
+
+FROM mcr.microsoft.com/dotnet/sdk:7.0 AS build
+ENV SENTRY_AUTH_TOKEN=${SENTRY_AUTH_TOKEN}
+ENV SENTRY_ORG=${SENTRY_ORG}
+ENV ASPNETCORE_SENTRY_PROJECT=${ASPNETCORE_SENTRY_PROJECT}
```

Comment:
FYI - I hate that we have to do this - especially for the auth token.  Ideally we'd not have to templatize the dockerfile at all, and would just use `ARG` to pass through to `docker build` ([see this blog for details](https://vsupalov.com/docker-build-pass-environment-variables/)).  Unfortunately, GAE doesn't let us control the `docker build` command it uses.  There's no way to pass the arguments in.  ([see stackoverflow question and answer](https://stackoverflow.com/q/68152233/634824)).

Reply:
Note, `env_variables` set in the `app.yaml` are passed into the docker container only at run time - in the `final` stage of this file.  They're not available during the `build` stage, which is where we need them.  (This is for getting the `dotnet build` to invoke Sentry CLI to upload symbols and sources.)